pkg/actuators/awsplacementgroup/controller.go

```diff
@@ -1,93 +0,0 @@
-package awsplacementgroup
```

Comment:
Since we are removing the `"github.com/openshift/api/machine/v1".AWSPlacementGroup` type from `api`, if we keep the controller code around (even if not running/stopped) we will still need to provide the type in some way otherwise IDEs, linters and `go test` will complain about not finding references for it with:
```
# github.com/openshift/machine-api-provider-aws/pkg/actuators/awsplacementgroup
pkg/actuators/awsplacementgroup/controller.go:30:8: undefined: "github.com/openshift/api/machine/v1".AWSPlacementGroup
pkg/actuators/awsplacementgroup/controller.go:48:24: undefined: "github.com/openshift/api/machine/v1".AWSPlacementGroup
pkg/actuators/awsplacementgroup/controller.go:89:92: undefined: "github.com/openshift/api/machine/v1".AWSPlacementGroup
```

Reply:
I would suggest copying the API to this repo for now and replacing the import statements

"github.com/openshift/api/machine/v1" > "github.com/openshift/machine-api-provider-aws/pkg/api/machine/v1"

Reply:
Ok sounds reasonable 👍